src/main/java/eu/diceh2020/jenkinsci/plugins/diceqt/DiceQTResultArchiver.java

```diff
@@ -155,6 +150,15 @@ public static void archiveResults(Run<?, ?> build, FilePath workspace,
 		} else {
 			build.addAction(action);
 		}
+
+		// Also save the project action;
+		DiceQTResultProjectAction projectAction = build.getAction(
```

Comment:
“Project action” is a misleading term here, since it is attached to a `Run` not a `Job`. In fact it is unclear why you are attaching two `Action`s to the same `Run` to begin with, the second of which has no apparent data in it. Just to create two sidebar links? There is a better way to do that (override the `actions` view IIRC).